Hadrons/Modules/MScalar/FreeProp.cpp

```diff
@@ -39,32 +40,29 @@ TFreeProp::TFreeProp(const std::string name)
 {}
 
 // dependencies/products ///////////////////////////////////////////////////////
-std::vector<std::string> TFreeProp::getInput(void)
+std::vector<std::string> TFreeProp::getInput()
 {
-    std::vector<std::string> in = {par().source};
-    
-    return in;
+    return {par().source};
```

Comment:
We should keep `void` in the argument signature and keep `in` as a separate `std::vector<std::string>` for consistency with the rest of Hadrons here.